app/controllers/idv/personal_key_controller.rb

```diff
@@ -30,7 +30,9 @@ def update
         fraud_review_pending: fraud_review_pending?,
         fraud_rejection: fraud_rejection?,
       )
-      idv_session.personal_key = nil
+
+      idv_session.acknowledge_personal_key
```

Comment:
Since it changes the session state, do we want to add a bang?
```suggestion
      idv_session.acknowledge_personal_key!
```

Reply:
Yeah I think that's a good call!